src/yxy.rs

```diff
@@ -60,7 +60,7 @@ impl<T: Float> Alpha<Yxy<T>, T> {
 
 impl<T: Float> FromColor<T> for Yxy<T> {
     fn from_xyz(xyz: Xyz<T>) -> Self {
-        let mut yxy = Yxy{ luma: xyz.y, ..Default::default() };
+        let mut yxy = Yxy{ x: T::zero(), y: T::zero(), luma: xyz.y };
```

Comment:
Is XYZ(0, 0, 0.0001) even a valid value? I had the impression that it's basically black if Y is 0.